packages/utils/src/stacktrace.ts

```diff
@@ -15,6 +22,26 @@ export function createStackParser(...parsers: StackLineParser[]): StackParser {
   return (stack: string, skipFirst: number = 0): StackFrame[] => {
     const frames: StackFrame[] = [];
 
+    for (const parser of sortedParsers) {
+      let debugIdCache = debugIdParserCache.get(parser);
+      if (!debugIdCache) {
+        debugIdCache = new Map();
+        debugIdParserCache.set(parser, debugIdCache);
```

Comment:
m: This is unbounded, which is not a problem for browser js, but it is for node. We may have re-think this as a result, or add some crude eviction mechanism.

Reply:
As discussed: The upper bound is the number of modules that have been injected with the debug id times the amount of stack trace parsers. I am gonna go ahead and say this is safely cachable.